src/servidor/UsuarioService.ts

```
@@ -14,5 +14,15 @@ class UsuarioService extends BaseService {
     return this.put<Usuario, Partial<Usuario>>(data, "/meu-perfil")
   }
 
+  async buscarPorEmail(email: string): Promise<Usuario | null> {
+    try {
+      return await this.get<Usuario>(`/buscar-email/${email}`)
+    } catch (error) {
+      if (typeof error === 'object' && error !== null && 'response' in error && (error as any).response?.status === 404) {
```

ESLint

[error] Replace `typeof·error·===·'object'·&&·error·!==·null·&&·'response'·in·error·&&·(error·as·any).response?.status·===·404` with `⏎········typeof·error·===·'object'·&&⏎········error·!==·null·&&⏎········'response'·in·error·&&⏎········(error·as·any).response?.status·===·404⏎······`

(prettier/prettier)